src/main/java/seedu/duke/main/CreateActivityFactor.java

```diff
@@ -0,0 +1,55 @@
+package seedu.duke.main;
+
+import seedu.duke.data.profile.Profile;
+import seedu.duke.data.profile.attributes.ActivityFactor;
+import seedu.duke.data.profile.utilities.ProfileUtils;
+import seedu.duke.logic.parser.exceptions.ParamMissingException;
+import seedu.duke.ui.Ui;
+
```

Comment:
Remember to write JavaDocs for your classes

Reply:
Thanks for reminding! will do it few hours later.